Translators/RoadConditions/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.10-slim
+
+ENV PYTHONUNBUFFERED TRUE
+
+ENV WORKDIR /app
+
+WORKDIR $WORKDIR
+
+COPY . .
+
+ENV FLASK_APP app/main.py
+
+RUN apt-get update && apt-get install -y libgeos-dev
+
+RUN pip install -r requirements.txt
```

Comment:
(non-blocking) If the required packages are unlikely to change frequently, we can reduce the image build time by moving these instructions above the `COPY . .` command. Currently, any changes to the files invalidate the Docker cache, causing the packages to be reinstalled instead of being retrieved from the cache.

Reply:
I've reordered the steps in the Dockerfile to run these before the COPY command as the required packages are unlikely to change frequently.